### PR TITLE
GSoC`22 report by omkar

### PR DESCRIPTION
**Pull request by OmkarPh**

### CODE REVIEW

**Comment by OmkarPh** [comment]
I've deployed my fork here for testing - https://test-swb-omkar.readthedocs.io/en/latest/index.html

i've put gsoc report page under `Contribute` section
https://test-swb-omkar.readthedocs.io/en/latest/contribute/gsoc_22.html

File: docs/source/contribute/gsoc_22.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Goals achieved
+-------------------
+
+* Refactor Workbench from vanilla javascript to React + Typescript implementation
+* Update Workbench to support the latest scancode-toolkit output format. (currently supports outputs from scancode-toolkit v31.1.1)
+* Improve User experience in Tableview, Home page, etc
+* Add convenience features like Drop files, History records, etc.
+* Add new sections for Scan Info, package info, etc
+
```
Comment (AyanSinhaMahapatra) [inline docs/source/contribute/gsoc_22.rst:28]: You can also mention the automated release scripts, those were very important!

File: docs/source/contribute/gsoc_22.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+Quick look
+-------------------
+.. image:: https://user-images.githubusercontent.com/48476025/185114433-7340ee70-57cd-4586-b7e7-964fe736451a.gif
+
+
+Links
+---------------
+* `Download ScanCode Workbench <https://github.com/nexB/scancode-workbench/releases/latest>`_
```
Comment (AyanSinhaMahapatra) [inline docs/source/contribute/gsoc_22.rst]: We can also add https://github.com/OmkarPh/scancode-workbench/releases/tag/v4.0.0betaPowershell2  and let's make sure we merge your code with all the latest changes when you think it's ready, so we have a release here, in the official repo. But we need not hurry, this is okay.

**Review by AyanSinhaMahapatra — APPROVED** [review]
Thanks, this looks great!

File: docs/source/contribute/gsoc_22.rst (hunk 1 above)
Comment (OmkarPh) [reply docs/source/contribute/gsoc_22.rst:28]: yeah, sure

File: docs/source/contribute/gsoc_22.rst (hunk 2 above)
Comment (OmkarPh) [reply docs/source/contribute/gsoc_22.rst]: so you mean, lets change that link to release in my fork, and when official release is done, we''ll update this right

Comment (AyanSinhaMahapatra) [reply docs/source/contribute/gsoc_22.rst]: No let's keep both, no need to come back and update.

**Comment by AyanSinhaMahapatra** [comment]
@OmkarPh We decided to have all the GSoC reports at https://aboutcode.readthedocs.io/en/latest/archive/gsoc-toc.html actually. Since you had these PR here, I've added your report there already. But if you want to change anything (now or later) feel free to open a PR there instead.

**Comment by OmkarPh** [comment]
Ohh great

So, I should put this link in evaluation right https://aboutcode.readthedocs.io/en/latest/archive/gsoc/reports/2022/scancode_workbench_omkar.html

**Comment by AyanSinhaMahapatra** [comment]
Thanks! I'll close this as it's in the aboutcode docs.

**Pull request closed without merging**